libraries/Aws/ServiceHealth/service_health.py

```diff
@@ -0,0 +1,88 @@
+from itertools import chain, product
+from datetime import datetime
+import requests
+import yaml
+from RW.Utils import parse_timedelta, csv_to_list
```

Comment:
I'll do the publishing asap (ideally tonight) and post the URL back here for you to pip install from

Reply:
no need to hurry :)

Reply:
https://pypi.org/project/runwhen-keywords/

Reply:
added